examples/stable/mysql/mysql-blueprint.yaml

```diff
@@ -45,8 +45,13 @@ actions:
           name: '{{ index .Object.metadata.labels "release" | toString }}'
           namespace: '{{ .StatefulSet.Namespace }}'
       args:
+<<<<<<< HEAD
         image: kanisterio/mysql-sidecar:0.40.0
         namespace: "{{ .StatefulSet.Namespace }}"
+=======
+        image: kanisterio/mysql-sidecar:0.41.0
+        namespace: "{{ .Deployment.Namespace }}"
+>>>>>>> master
         command:
```

Comment:
@ankitjain235 
something wrong here?

Reply:
Accidentally missed this merge while rebasing with master. Fixed in the last commit.